website/docs/r/autoscale_setting.html.markdown

```diff
@@ -0,0 +1,351 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_autoscale_setting"
+sidebar_current: "docs-azurerm-resource-autoscale-setting"
+description: |-
+  Create an autoscale setting.
+---
+
+# azurerm\_autoscale\_setting
+
+Create an autoscale setting.
+
+## Example Usage
+
+```hcl
+variable "time_zone" {
+  type    = "string"
+  default = "Pacific Standard Time"
+}
+
+variable "adminPwd" {
+  type    = "string"
+  default = "^l&f_ZwX2L4C"
+}
+
+resource "azurerm_resource_group" "test" {
+  name     = "autoscalingTest"
+  location = "West US"
+}
+
+resource "azurerm_autoscale_setting" "test" {
+  name                = "myAutoscaleSetting"
+  enabled             = true
+  resource_group_name = "${azurerm_resource_group.test.name}"
+  location            = "${azurerm_resource_group.test.location}"
+  target_resource_uri = "${azurerm_virtual_machine_scale_set.test.id}"
```

Comment:
In Terraform we tend to use `id` rather than `uri`/`url` - so I'd suggest it'd be better to change this to match the convention?